components/PostNewsletter/PostNewsletter.jsx

```diff
@@ -20,10 +21,12 @@ const PostNewsletter = ({
 
   const [isMessageHidden, setMessageHidden] = useState(true)
   const withoutSubscribeForm = language !== 'ru' || type === 'news'
```

Comment:
Давай вот тут тогда под один вид приведём: 
```suggestion
  const withSubscribeForm = language === 'ru' || type !== 'news'
```
И ниже на 32 строке заменим `!withoutSubscribeForm` на `withSubscribeForm`. 
Лучше всегда стараться в условиях писать булевые значения без `!`, такой код проще читать и поддеживать, потому что убирая оператор `не` мы уменьшаем количество операций на 1

Reply:
Переделал